src/mdlComponentHandler.js

```diff
@@ -86,6 +86,15 @@ var componentHandler = (function() {
    * the element to.
    */
   function upgradeElementInternal(element, jsClass) {
+    if (jsClass === undefined) {
+      for (var i = 0; i < registeredComponents_.length; i++) {
```

Comment:
Ah, when I originally wrote this handler I was aiming for IE9+. Won't a lot of people still need IE9 support? This concerns me.